docs/events.rst

```diff
@@ -74,7 +78,9 @@ It's also possible to dump the raw events using the API.
        }
     ]
 
-The ``set`` parameter accepts an optional ``union`` or ``intersection`` argument to determine the behavior for filtering sets of tags (i.e. inclusive or exclusive). By default, Graphite uses a "lazy union" that will return any matching events for a given tag in a list of tags. This behavior is not intuitive and will therefore be deprecated in a future release.
+The ``set`` parameter accepts an optional ``union`` or ``intersection`` argument to determine the behavior for filtering sets of tags (i.e. inclusive or exclusive). By default, Graphite uses a "lazy union" that will return any matching events for a given tag in a list of tags. This behavior is not intuitive and will therefore be deprecated in a future release. When matching using multiple tags, separate the tags with a space.
+
+*Note*: Prior to 0.10.0, the value of ``tags`` will be a string, with multiple tags being separated by a space.
```

Comment:
Ok, this still needs to be cleaned up. I really dislike that sentence being slapped onto the `set` paragraph, it doesn't belong there. And I don't think we need that *Note* since it's referring to the output. Nowhere else do we explicitly document the format of each key/value output. As long as we have an accurate example (which we do, [here](https://github.com/graphite-project/graphite-web/blame/master/docs/events.rst#L56-L75)), I don't see the point.

TL;DR please remove the changes in this snippet.